var/spack/repos/builtin/packages/wcslib/package.py

```diff
@@ -47,8 +47,19 @@ def configure_args(self):
 
         return args
 
+    def headers(self):
+        return find_headers('wcs', self.prefix.include, recursive=True) \
+          or None  # return None to indicate failure
+
     @run_after('install')
     def darwin_fix(self):
         # The shared library is not installed correctly on Darwin; fix this
         if self.spec.satisfies('platform=darwin'):
             fix_darwin_install_name(self.prefix.lib)
+
+    @run_after('install') #Is this duplicate necessary?
+    def symlink_include(self):
+        os.symlink(
+            os.path.join(self.prefix, 'include', 'wcslib'),
+            os.path.join(self.prefix, 'include')
+        )
```

Comment:
I don't love this solution. Packages should always store their headers in a subdirectory, and packages that depend on them should respect this. By putting everything in `self.prefix.include`, you're risking conflicting files when multiple packages are merged to the same directory.

I'm going to investigate this further. Maybe Astropy 4.0 fixes this problem. Or a different version of wcslib is required. I'm also seeing issues with `berkeley-db` though, at least on macOS.

Reply:
I think the right solution is to add a build dependency on pkg-config. I'm also adding wcslib 7.3 and astropy 4.0, so I'll make new PRs for these.